content_scripts/bullet-screens.jsx

```diff
@@ -17,9 +18,14 @@ function makeBulletApp(root, { material, onOperate, destroy, autoAudio, bulletSp
 
     useEffect(() => {
       const listenEnter = () => {
-        if (audioRef.current) {
-          audioRef.current.play();
-        }
+        audioRef?.current?.play()?.then(() => setNeedFirstClickHint(false))?.catch(e => {
```

Comment:
其实我们可以直接在Github的PR页面修改的，你可以点击指定的代码行提出修改之要求。

比如：

```suggestion
        audioRef!.current?.play()?.then(() => setNeedFirstClickHint(false))?.catch(e => {
```

待会还有个新功能或许需要好好讨论一番，届时就用Github自带的各项讨论功能吧。